pkg/api-server/zones_ws.go

```diff
@@ -52,7 +52,7 @@ func toZones(rlist system.ZoneOverviewResourceList) Zones {
 	for _, overview := range rlist.Items {
 		zones = append(zones, Zone{
 			Name:   overview.GetMeta().GetName(),
-			Active: overview.Spec.ZoneInsight.IsOnline(),
+			Active: overview.Spec.GetZoneInsight().IsOnline() && overview.Spec.GetZone().GetEnabled(),
 		})
 	}
 	return zones
```

Comment:
Is this API still used somewhere?

Reply:
If I get it right it is used here:
![image](https://user-images.githubusercontent.com/12110261/102902564-bf9b9500-44a1-11eb-9331-677ae7a18c81.png)
Is it correct?

Reply:
Checked the GUI code. Yeah, for some reason it is used. I was pretty sure we want to drop this in favor of insights.

I think we should have a column called enabled / disabled. Putting zone offline when we disable it is not exactly correct.

Let's keep it for now because I don't think it matters, but create a card for getting rid of this endpoint and changing the GUI